docs/application/web/guides/machine-learning/single.md

```diff
@@ -10,7 +10,7 @@ The main features of the Single API include:
 
 - Invoking the neural network model
 
-  After creating the `SingleShot` instance, you can [invoke the model](#invoke) with the input data and get the inference output result.
+  After creating the `SingleShot` instance, you can invoke the model [synchronously](#invoke) or [asynchronously](#invoke-asynchronously) with the input data and get the inference output result.
```

Comment:
Kindly ignore my previous comment, since I checked the "Master" branch for content, instead of checking in "tizen_6.5_prepare" branch. Sorry for the confusion.